web/client/epics/identify.js

```diff
@@ -58,6 +59,58 @@ const stopFeatureInfo = state => stopGetFeatureInfoSelector(state) || isFeatureG
 
 import {getFeatureInfo} from '../api/identify';
 
+/**
+ * Recalculates pixel and geometric filter to allow also GFI emulation for WFS.
+ * This information is used also to switch to edit bode from GFI applying the same filter
+ * @param {object} point the point clicked, emitted by featureInfoClick action
+ * @param {string} projection map projection
+ */
+const updatePointWithGeometricFilter = (point, projection) => {
+    // calculate a query for edit
+    const lng = get(point, 'latlng.lng');
+    const lat = get(point, 'latlng.lat');
+    // update pixel if changed
+    const pos = reproject([lng, lat], 'EPSG:4326', projection);
+    const getPixel = getHook(GET_PIXEL_FROM_COORDINATES_HOOK);
+    let pixel;
+    if (getPixel) {
+        const [x, y] = getPixel([pos.x, pos.y]);
+        pixel = { x, y };
+    } else {
+        pixel = point.pixel;
+    }
+    const hook = getHook(GET_COORDINATES_FROM_PIXEL_HOOK);
+    const radius = calculateCircleRadiusFromPixel(
+        hook,
+        pixel,
+        pos,
+        5
+    );
+    // emulation of feature info filter to query WFS services (edit and/or WFS layer)
+    // TODO: evaluate to do this calculation at map level, so we can use this filter also in popups
```

Comment:
i think i'm seeing here some tech debt :)

Reply:
This comment should be removed, because I noticed that we can not do it (the update of coordinates, for instance, need s to realculate the geometric filter anyway. I'll remove it
```suggestion
```